packages/manager/src/factories/cloudpulse/alerts.ts

```diff
@@ -29,6 +32,42 @@ export const alertRulesFactory = Factory.Sync.makeFactory<AlertDefinitionMetricC
 );
 
 const alertTypes: AlertDefinitionType[] = ['system', 'user'];
+export const triggerConditionFactory = Factory.Sync.makeFactory<TriggerCondition>(
+  {
+    criteria_condition: 'ALL',
+    evaluation_period_seconds: 300,
+    polling_interval_seconds: 60,
```

Comment:
```suggestion
    polling_interval_seconds: 600,
```
CPU metric scrape interval is '2m' , so as per the filtering logic in TriggerConditions, it won't show the `1min` as an option .